Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -1008,15 +1008,17 @@ extension MapboxMap {
 
 extension MapboxMap {
     /// Errors related to MapProjection API
+    @available(iOS, obsoleted: 11.0)
     @_spi(Experimental) public enum MapProjectionError: Error {
         case unsupportedProjection
     }
```

Comment:
What's the benefit of `obsoleted` vs just deleting it?

Reply:
Well spotted, I committed this accidentally after experimenting with `deprecated` vs `obsoleted` for these symbols. Changed it back to deprecated.